src/components/plots/styling/PlotStyling.jsx

```diff
@@ -86,7 +86,7 @@ const PlotStyling = (props) => {
   });
 
   return (
-    <Collapse accordion defaultActiveKey={defaultActiveKey}>
+    <Collapse accordion defaultActiveKey={defaultActivePanel}>
```

Comment:
`defaultActiveKey` is the name given by antd, I think it will be less confusing to preserve that name instead of changing to `defaultActivePanel` and using 2 different names for the same thing.

Reply:
I think semantically `defaultActivePanel` is more meaningful, as it's easier to understand that we want the `defaultActivePanel` to be the one that's opened first. or `defaultActivePanelKey`?